content/docs/1.6.0/references/settings.md

```diff
@@ -795,7 +819,7 @@ Longhorn system contains user deployed components (e.g, Longhorn manager, Longho
 You need to set node selector for both of them. This setting only sets node selector for system managed components. Follow the instruction at [Node Selector](../../advanced-resources/deploy/node-selector) to change node selector.
 
 > **Warning:**  Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.
-Make sure all Longhorn volumes are `detached`. If there are running Longhorn volumes in the system, this means the Longhorn system cannot restart its components and the request will be rejected.
+Ensure that all Longhorn volumes are detached. When volumes are in use, Longhorn components are not restarted so you might be required to configure this setting again when all volumes are finally detached.
```

Comment:
Why need to ensure all volumes are detached? Also, don't use `might be required`, as this would be unclear for users. Whether they need to reconfigure or not, this should be very clear. 

You can briefly explain the function and when it will be taken. For the details, refer to the advanced resource page. 

The same suggestion for other danger zone settings in this setting page.

Reply:
Updated to remove `might` and add a brief explanation.